scripts/remove

```diff
@@ -34,52 +34,124 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get $app domain)
 firmware_nonfree=$(ynh_app_setting_get $app firmware_nonfree)
+service_name=$(ynh_app_setting_get $app service_name)
+final_path=$(ynh_app_setting_get $app final_path)
+
+#=================================================
+# STANDARD REMOVE
+#=================================================
+# REMOVE SERVICE FROM ADMIN PANEL
+#=================================================
+
```

Comment:
With all those lock stories, I'm wondering if we shouldn't add a `yunohost service stop` here to make sure the service is properly stopped before going on ... To be checked if that's really needed...

Reply:
I don't get all the history, but I can stop it here yes